### ISSUE TITLE
document adding holidays in more detail

### ISSUE THREAD

**Issue by davewhiteland** [mysociety/alaveteli#2828]
Currently we have
http://alaveteli.org/docs/installing/next_steps/#add-some-public-holidays

but could be much more (screenshots by @garethrees in support resonse).

![screen shot 2015-10-07 at 09 30 16](https://cloud.githubusercontent.com/assets/111320/10335538/5f2fd20c-6ce9-11e5-98b1-d0aba9c61c1b.png)

**Comment by garethrees** [comment]
TBH I think this interface is one of the more self explanatory ones. Surprised I had to explain it. Any ideas on whether we could make it clearer from this page?

![screen shot 2015-10-07 at 09 25 37](https://cloud.githubusercontent.com/assets/282788/10335619/f444a124-6ce9-11e5-9a14-408380094f7c.png)

**Comment by davewhiteland** [comment]
Yeah I'm loathe to put _too_ many screenshots in the docs where we don't need them simply because that's a monkey for our backs when the interfaces changes... but when I come back to this I'll have a go at adding some holidays myself and see how much of a mess I can make of it...  And then it may indeed be more a case for interface rather than doc work.

**Comment by garethrees** [comment]
Related #2800

**Comment by RichardTaylor** [comment]
As an administrator coming to update the holidays I wondered where the "built-in suggestions" came from and if I could trust them.

**Comment by RichardTaylor** [comment]
For the England and Wales in 2021 the "built in Suggestions" omitted Easter Monday and Summer bank holiday,

**Comment by garethrees** [comment]
> As an administrator coming to update the holidays I wondered where the "built-in suggestions" came from and if I could trust them.

They come from the [holidays](https://github.com/holidays/holidays) ruby library that we [include](https://github.com/mysociety/alaveteli/blob/0.38.3.0/Gemfile#L108) in Alaveteli.

**Comment by garethrees** [comment]
Noting that we have some [WDTK documentation](https://wdtkwiki.mysociety.org/wiki/Public_Holidays) on this.

**Comment by HelenWDTK** [comment]
This issue has been automatically closed due to a lack of discussion or resolution for over 12 months.
Should we decide to revisit this issue in the future, it can be reopened.
